src/msg/Message.cc

```diff
@@ -79,7 +79,7 @@ using namespace std;
 #include "messages/MOSDPGTrim.h"
 #include "messages/MOSDPGMissing.h"
 #include "messages/MOSDScrub.h"
-#include "messages/MOSDRepScrub.h"
+#include "messages/MOSDBuildScrubMap.h"
```

Comment:
@athanatos I just wanted to make the code change clearer.  You didn't suggest it.